.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+-   id: isort
+    name: isort
+    entry: isort
+    require_serial: true
```

Comment:
Hi, thanks for this PR @gurunath-p! Was there a reason you chose to use `require_serial` instead of allowing pre-commit to parallelize isort? Just curious if there was a discovered gotcha. If not, we can try resuming parallel mode on `develop`.